planner/planner_API.go

```diff
@@ -441,7 +441,7 @@ func (planner MyPlanner) SetupRouter(serverPort string) *http.Server {
 	}
 
 	svr := &http.Server{
-		Addr:         ":" + serverPort,
+		Addr:         "0.0.0.0:" + serverPort,
```

Comment:
Would this work for Heroku?

Reply:
it shoud, do you think we can just deploy the branch and try out

Reply:
I deployed a testing app: https://dashboard.heroku.com/apps/vacation-pla-feature-ad-epm8rp